src/script/components/DeviceCard.test.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Wire
+ * Copyright (C) 2021 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import ko from 'knockout';
+import {ClientClassification} from '@wireapp/api-client/src/client';
+import TestPage from 'Util/test/TestPage';
+import DeviceCard, {DeviceCardProps} from './DeviceCard';
+import type {ClientEntity} from '../client/ClientEntity';
+
+class DeviceCardPage extends TestPage<DeviceCardProps> {
+  constructor(props?: DeviceCardProps) {
+    super(DeviceCard, props);
+  }
+
+  getDesktopIcon = () => this.get('svg[data-uie-name="status-desktop-device"]');
+  getMobileDeviceIcon = () => this.get('svg[data-uie-name="status-mobile-device"]');
+}
+
+describe('DeviceCard', () => {
```

Comment:
Let's add some more tests for:
it('shows disclose icon when component is clickable', ...)
it('shows verified icon', ...)
it('shows unverified icon', ...)

Reply:
done in https://github.com/wireapp/wire-webapp/pull/10436/commits/3ddd48e372f301af5a330484fe80fa8e184536af